app/shared/src/commonMain/kotlin/ui/subject/components/comment/CommentLoader.kt

```diff
@@ -47,10 +49,40 @@ class CommentLoader<T>(
     }
 
     companion object {
+        /**
+         * create comment loader of subject
+         */
+        fun createForSubject(
+            subjectId: Flow<Int>,
+            coroutineContext: CoroutineContext,
+            subjectCommentSource: suspend LazyDataCacheContext.(Int) -> PagedSource<SubjectComment>
+        ) = CommentLoader(
+            source = subjectId.map { sid ->
+                LazyDataCache(
+                    createSource = { subjectCommentSource(sid) },
+                    getKey = { it.id },
+                    debugName = "subjectComment-$sid",
+                )
+            },
+            coroutineContext = coroutineContext,
+            uiMapper = { comment ->
+                UIComment(
+                    id = comment.id,
+                    creator = comment.creator ?: UserInfo.EMPTY,
```

Comment:
```suggestion
                    creator = comment.creator,
```

use null incase of absent